app/_js/directives.js

```diff
@@ -147,7 +147,7 @@ angular.module('wmProfile.directives', [])
       return {
         restrict: 'E',
         scope: {
-          lang: $rootScope.lang,
+          lang: '=lang',
```

Comment:
should be `lang: '='`. you don't need to repeat the variable name if it's the same